L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -99,6 +99,18 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         }
     }
 
+    func tapKeyboardKey(_ key: Int) {
+        //let key = self.keyboards.buttons[key]
```

Comment:
@clarmso please remove this comment, it is not needed and Swiflint is complaining about it:
`Prefer at least one space after slashes for comments`